prooftools/representationproof_test.go

```diff
@@ -0,0 +1,142 @@
+package prooftools_test
+
+import (
+	"testing"
+
+	"github.com/privacybydesign/gabi/big"
+	"github.com/privacybydesign/gabi/keyproof"
+	"github.com/privacybydesign/gabi/prooftools"
+	"github.com/stretchr/testify/assert"
+)
+
+type RepTestSecret struct {
+	secrets     map[string]*big.Int
+	randomizers map[string]*big.Int
+}
+
+func (rs *RepTestSecret) Secret(name string) *big.Int {
+	res, ok := rs.secrets[name]
+	if ok {
+		return res
+	}
+	return nil
```

Comment:
Did you do this ok check on purpose for clarity? It is not necessary; replacing the function body with just `return rs.secrets[name]` would behave the same. Same goes for similar functions below.

Reply:
fixed